### ISSUE TITLE
volume prune only supported with daemon version >= 1.25

### ISSUE THREAD

**Issue by helli0n** [docker/classicswarm#2709]
```
#docker -H tcp://localhost:4000 volume prune
only supported with daemon version >= 1.25
```
```
# docker -H tcp://localhost:4000 images
REPOSITORY                           TAG                 IMAGE ID            CREATED             SIZE
swarm                                1.2.6               36b1e23becab        4 months ago        15.9 MB
swarm                                latest              36b1e23becab        4 months ago        15.9 MB
```

```
# docker -H tcp://localhost:4000 ps -a 
CONTAINER ID        IMAGE                                     COMMAND                  CREATED             STATUS                    PORTS                                                                                                                                        NAMES 
 e0cad4150c32        swarm                                     "/swarm --debug ma..."   10 days ago         Up 7 days                 2375/tcp, 10.1.1.1:4000->4000/tcp                                                                                                           localhost/swarm-manager
```

Please update Docker Swarm Container

**Comment by allencloud** [comment]
I think it is reasonable to add prune. WDYT? @dongluochen @nishanttotla

**Comment by nishanttotla** [comment]
I'll have to look up, on which version prune was added to the API. Do you mean we should update the old version container?

**Comment by dongluochen** [comment]
Yes I think it reasonable to add prune. We should add image, container, volume prune one by one. And validate Swarm manager's cache is up-to-date after prune operations.

**Comment by dongluochen** [comment]
We are going to release 1.2.7 in next few weeks. I think prune operations can come in after that.

**Comment by helli0n** [comment]
Hello.
I have updated swarm to 1.2.8 and tried to do **volume prune**

```
#docker -H tcp://localhost:4000 volume prune
only supported with daemon version >= 1.25
```

**Comment by allencloud** [comment]
Hi， @helli0n 
1.2.8 comes to fix a problem of 1.2.7.

I am afraid this will come after 1.2.8. And Actually I am working on this.

**Comment by helli0n** [comment]
ok, I'll wait.
thanks
